src/main/java/micdoodle8/mods/galacticraft/planets/asteroids/nei/NEIGalacticraftAsteroidsConfig.java

```diff
@@ -156,9 +152,9 @@ public void registerRecipes()
         input.add(new PositionedStack(new ItemStack(GCItems.heavyPlatingTier1), 116 - x, 37 - y));
         input.add(new PositionedStack(new ItemStack(GCItems.heavyPlatingTier1), 116 - x, 55 - y));
         input.add(new PositionedStack(new ItemStack(GCItems.heavyPlatingTier1), 116 - x, 73 - y));
-        input.add(new PositionedStack(new ItemStack(IronChest.ironChestBlock, 1, 1), 80 - x, 55 - y));
-        input.add(new PositionedStack(new ItemStack(IronChest.ironChestBlock, 1, 1), 98 - x, 55 - y));
-        input.add(new PositionedStack(new ItemStack(AsteroidsItems.basicItem, 1, 8), 44 - x, 73 - y));
```

Comment:
Are you sure this is a chest item ? new ItemStack(AsteroidsItems.basicItem, 1, 8)

Reply:
input.add(new PositionedStack(new ItemStack(AsteroidsItems.basicItem, 1, 8), 44 - x, 73 - y)); is the beam core

Reply:
Fixed.